docker-run.sh

```diff
@@ -44,8 +44,9 @@ sendmail_path = /usr/sbin/sendmail -t -i
 memory_limit = ${PHP_INI_MEMORY_LIMIT}
 EOF
 
-  if [[ ! -f /var/www/html/conf/conf.php ]]; then
+if [[ ! -f /var/www/html/conf/conf.php ]]; then
     echo "[INIT] => update Dolibarr Config ..."
+    auth=${DOLI_AUTH:-dolibarr}
```

Comment:
prefer UPPER bash var names, better to make difference while readiing.
```suggestion
    AUTH=${DOLI_AUTH:-dolibarr}
```

Reply:
You can use directly ${DOLI_AUTH} whose default value is defined on Dockerfile.

Reply:
I think so ^_^